src/Program.fs

```diff
@@ -583,9 +609,10 @@ let generate (configFile: string) =
                 colorprintfn "✏️  Generating Fable $green[%s]" sharedFableDocument
 
                 let packageReferences = [
-                    MSBuildXElement.PackageReferenceUpdate("FSharp.Core", "4.7.2")
+                    MSBuildXElement.PackageReferenceUpdate("FSharp.Core", "5.0.1")
                     MSBuildXElement.PackageReferenceInclude("Fable.SimpleHttp", "3.0.0")
                     MSBuildXElement.PackageReferenceInclude("Fable.SimpleJson", "3.21.0")
+                    MSBuildXElement.PackageReferenceInclude("Newtonsoft.Json", "12.0.2")
```

Comment:
Same as above, Newtonsoft.Json isn't needed in Fable projects, that what `Fable.SimpleJson` is for